lib/timeline/component/item/BackgroundItem.js

```diff
@@ -105,16 +99,73 @@ BackgroundItem.prototype.redraw = function() {
     // update class
     var className = (this.data.className ? (' ' + this.data.className) : '') +
         (this.selected ? ' vis-selected' : '');
-    dom.box.className = this.baseClassName + className;
+    this.dom.box.className = this.baseClassName + className;
+  }
+}
+BackgroundItem.prototype._getDomComponentsSizes = function() {
+  // determine from css whether this box has overflow
+  this.overflow = window.getComputedStyle(this.dom.content).overflow !== 'hidden';
+  return {
+    content: {
+      width: this.dom.content.offsetWidth
+    }
+  }
+}
+BackgroundItem.prototype._updateDomComponentsSizes = function(sizes) {
```

Comment:
Separate by empty line. I personally prefer two empty lines between functions. This comment applies to the whole file.